src/NuGet.Clients/NuGet.CommandLine/Commands/ProjectFactory.cs

```diff
@@ -242,6 +242,11 @@ public PackageBuilder CreateBuilder(string basePath, NuGetVersion version, strin
             {
                 ExtractMetadata(builder);
             }
+            catch (PackagingException packex) when (packex.AsLogMessage().Code.Equals(NuGetLogCode.NU5133))
+            {
+                Logger.Log(PackagingLogMessage.CreateError(packex.Message, NuGetLogCode.NU5133));
```

Comment:
Use ExceptionUtilities that way you don't have to create a PackagingLogMessage.

Reply:
I changed to `ExceptionUtilities.LogException(packex, Logger);`
But exception message with verbosity detailed is quite large. Is this we really need to show to customers?
<img width="1328" alt="image" src="https://user-images.githubusercontent.com/8766776/154608814-7be89863-87ca-46d7-a636-b6639dadfdfd.png">

Reply:
@nkolev92 
Is it ok to show above detailed info to end user? My concern is how relevant it would be to them since we already informed them what is going on.

Reply:
Resolving. It looks we do show very detailed exception for other cases too.